walker_lake/walker_lake.ipynb

```diff
@@ -131,15 +135,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(\"The shape of the DataArray is :\", landsat_5_img.shape)\n",
```

Comment:
Why were these print statements removed? The idea is to give the user some information about what is loaded in a friendly way...

Reply:
I removed it because the xarray repr displays that information nicely too. Maybe it wasn't the case when the original example was created? I can add it back it you prefer!

![image](https://github.com/user-attachments/assets/86bcd889-5b9d-4a58-ad22-625248df73a1)

Reply:
Yes, I don't think it had such a fancy repr back then!